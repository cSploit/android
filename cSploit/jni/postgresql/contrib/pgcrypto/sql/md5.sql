--
-- MD5 message digest
--

SELECT encode(digest('', 'md5'), 'hex');
SELECT encode(digest('a', 'md5'), 'hex');
SELECT encode(digest('abc', 'md5'), 'hex');
SELECT encode(digest('message digest', 'md5'), 'hex');
SELECT encode(digest('abcdefghijklmnopqrstuvwxyz', 'md5'), 'hex');
SELECT encode(digest('ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789', 'md5'), 'hex');
SELECT encode(digest('12345678901234567890123456789012345678901234567890123456789012345678901234567890', 'md5'), 'hex');
