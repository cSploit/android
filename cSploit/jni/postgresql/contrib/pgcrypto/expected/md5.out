--
-- MD5 message digest
--
SELECT encode(digest('', 'md5'), 'hex');
              encode              
----------------------------------
 d41d8cd98f00b204e9800998ecf8427e
(1 row)

SELECT encode(digest('a', 'md5'), 'hex');
              encode              
----------------------------------
 0cc175b9c0f1b6a831c399e269772661
(1 row)

SELECT encode(digest('abc', 'md5'), 'hex');
              encode              
----------------------------------
 900150983cd24fb0d6963f7d28e17f72
(1 row)

SELECT encode(digest('message digest', 'md5'), 'hex');
              encode              
----------------------------------
 f96b697d7cb7938d525a2f31aaf161d0
(1 row)

SELECT encode(digest('abcdefghijklmnopqrstuvwxyz', 'md5'), 'hex');
              encode              
----------------------------------
 c3fcd3d76192e4007dfb496cca67e13b
(1 row)

SELECT encode(digest('ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789', 'md5'), 'hex');
              encode              
----------------------------------
 d174ab98d277d9f5a5611c2c9f419d9f
(1 row)

SELECT encode(digest('12345678901234567890123456789012345678901234567890123456789012345678901234567890', 'md5'), 'hex');
              encode              
----------------------------------
 57edf4a22be3c955ac49da2e2107b67a
(1 row)

